.github/workflows/push-translations.yml

```
@@ -0,0 +1,46 @@
+name: Push Translations to Mojito
+
+on:
+  push:
+    paths:
+      - client-participation/js/strings/en.js
+    branches:
+      - 320-continuous-localization
```

Accept this only when ready to merge:

```suggestion
      - dev
```

For simplicities sake, we only try to localize strings on mainline. Mojito allows more complex workflows of localizing feature branches, but let's worry about that later, if at all.